spec/factories/movies.rb

```diff
@@ -6,21 +6,23 @@
 #  desc       :text
 #  position   :integer
 #  title      :string
-#  url        :string
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
+#  channel_id :string
 #  genre_id   :integer
 #  text_id    :integer
 #
 # Foreign Keys
 #
 #  fk_rails_...  (genre_id => genres.id)
 #
-
+Faker::Config.locale = :en
 FactoryBot.define do
   factory :movie do
     sequence(:title) { |n| "#{n}_#{Faker::University.name}" }
-    sequence(:url) { |n| "#{n}_#{Faker::Commerce.department}" }
-    sequence(:genre) { |n| "#{n}_#{Faker::Business.credit_card_type}" }
+    sequence(:channel_id) { |n| "#{n}_#{Faker::Address.city}" }
```

Comment:
雑ですがとりあえずこのあたりにしていただけますか。

```suggestion
    sequence(:channel_id) { |n| "https://www.youtube.com/watch?v=#{SecureRandom.hex(8)}z#{n}" }
```